static/engine/DynamicHtml.js

```diff
@@ -0,0 +1,86 @@
+export class DynamicHtml {
+
+  static selectedImage = "";
+
+  static handleSelectElement(data) {
+	  const select = createElement("select", ".footer .item", "selector");
+	  select.style.zIndex = 1
+	  appendLabelSelect(select);
+	  for (const key in data) {
+		  if (key == "path") {
+			  continue;
+		  }
+		  let element = createElement("option", select);
+		  element.innerHTML = key;
+	  }
+	  select.addEventListener("change", function () {
+		  var selectedValue = this.value;
+		  DynamicHtml.removeElement(this);
+		  if (selectedValue == "root") {
+			  DynamicHtml.createImageOption(data.root, data.path);
+		  } else {
+			  data[selectedValue].path = data.path + "/" + selectedValue;
+			  DynamicHtml.handleSelectElement(data[selectedValue]);
+		  }
+	  });
+  }
+
+  static handleImageOption(data, path) {
```

Comment:
what is data?
make data a better variable name

Reply:
changes to imageData